scripts/build-android.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Initialization.
+
+set -e
+
+# Preface.
+
+TOOLCHAIN="$PWD/../android-ndk-r19c/toolchain"
```

Comment:
Path not consistent with instructions: `--install-dir ./toolchain`

Reply:
Which instructions are referencing `--install-dir`?

Reply:
The standalone toolchain instructions from https://github.com/webmixedreality/exokit/pull/897#issuecomment-480236935

Reply:
Oh thanks, I think that comment just needs to be updated to mean the standalone toolchain should be boostrapped in android home dir, not this repo.

Reply:
Additionally, the right path forward is probably to globally require the `ANDROID_HOME` or `TOOLCHAIN` env var, so we don't deal in this resolution in our build scripts.

Reply:
This should work in that case:

`$ANDROID_HOME/ndk-bundle/build/tools/make_standalone_toolchain.py`

and then 
>  the standalone toolchain should be boostrapped in android home dir, not this repo
  

`$ANDROID_HOME/ndk-bundle`